contrib/ci-functional-tests.sh

```diff
@@ -27,4 +27,4 @@ cd tests/servers/cosignerd && cargo build && cd "$REPO_ROOT"
 python3 -m venv venv
 . venv/bin/activate
 pip install -r tests/requirements.txt
-VERBOSE=1 LOG_LEVEL=debug TIMEOUT=120 TEST_DEBUG=1 POSTGRES_USER="test" POSTGRES_PASS="test" pytest -n3 -vvv --log-cli-level=DEBUG tests/
+VERBOSE=1 LOG_LEVEL=debug TIMEOUT=120 TEST_DEBUG=1 POSTGRES_USER="test" POSTGRES_PASS="test" pytest -n3 -vvv --log-cli-level=DEBUG --timeout=600 tests/
```

Comment:
Ten minutes is not enough for the CI, we have pretty slow tests. `1800` seems reasonable: if it gets over this timeout, we are sure it's something with our tests not the CI being atrociously slow

Reply:
Just updated, I hoped that 10 mins were enough :(